app/src/main/java/com/onboarding/calculator/mvp/model/CalculatorModel.java

```diff
@@ -0,0 +1,85 @@
+package com.onboarding.calculator.mvp.model;
+
+import static com.onboarding.calculator.util.Util.ADD;
+import static com.onboarding.calculator.util.Util.CLEAN;
+import static com.onboarding.calculator.util.Util.DIV;
+import static com.onboarding.calculator.util.Util.EMPTY_STRING;
+import static com.onboarding.calculator.util.Util.EQUAL;
+import static com.onboarding.calculator.util.Util.MUL;
+import static com.onboarding.calculator.util.Util.OPERAND1;
+import static com.onboarding.calculator.util.Util.OPERAND2;
+import static com.onboarding.calculator.util.Util.SUB;
+
+public class CalculatorModel {
+
+    private String operator;
+    private String operand1;
+    private String operand2;
+    private int switchOp;
+
+    public CalculatorModel() {
+        reset();
```

Comment:
instead of call this method you can assign default values for the variables

Reply:
Solved!